package.json

```
@@ -184,7 +184,8 @@
             "fortran_fixed-form": "cd $dir && gfortran $fileName -o $fileNameWithoutExt && $dir$fileNameWithoutExt",
             "fortran": "cd $dir && gfortran $fileName -o $fileNameWithoutExt && $dir$fileNameWithoutExt",
             "sml": "cd $dir && sml $fileName",
-            "mojo": "mojo run"
+            "mojo": "mojo run",
+            "erlang": "cd $dir && escript $fileName"
```

Simplify with below:
```suggestion
            "erlang": "escript"
```